docs/openj9_support.md

```diff
@@ -43,15 +43,14 @@ columns will be removed over time.
+| 0.29.0         | Oct 2021      | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |                                                                                          |                                                                                      |                                                                                          |
+| 0.29.1         | Dec 2021      | :fontawesome-solid-times:{: .no aria-hidden="true"}<span class="sr-only">no</span>   | :fontawesome-solid-times:{: .no aria-hidden="true"}<span class="sr-only">no</span>   | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> (3) |                                                                                      |                                                                                          |
+| 0.30.0         | Jan 2022      | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     |                                                                                      |                                                                                          |
+| 0.32.0         | Apr 2022      | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |                                                                                          |
+| 0.33.0         | Jul 2022 (1)  | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |                                                                                          |
+| 0.34.0         | Sep 2022 (1)  | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-times:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> (2) |
```

Comment:
Only Java 19 should be supported in 0.34
8,11, 17, 18 should have a red x.
https://eclipse-openj9.github.io/openj9-docs-staging/922/openj9_support/

Reply:
Of course - I knew there were too many ticks in the table! Thanks for spotting that, it's now fixed.